images/cilium/tests/cilium-install.sh

```diff
@@ -14,29 +15,41 @@ if [ -z "$CI" ]; then
     trap "k3d cluster delete $CLUSTER_NAME" EXIT
 fi
 
+TMPDIR=$(mktemp -d --tmpdir=$SCRIPT_DIR)
+trap "rm -rf $TMPDIR" EXIT
```

Comment:
I thought there could only be on trap, so I'm a little concerned that this will leak k3s clusters.

Reply:
I've consolidated into one trap. Thanks